src/metabase/models/database.clj

```diff
@@ -136,6 +136,12 @@
     (catch Throwable e
       (log/error e "Error scheduling tasks for DB"))))
 
+(defn check-and-schedule-tasks!
+  "(Re)schedule sync operation tasks for any database which is not yet being synced regularly."
+  []
+  (doseq [database (t2/select Database)]
```

Comment:
nit - we're gradually moving towards using model keywords
```suggestion
  (doseq [database (t2/select :model/Database)]
```

Reply:
https://github.com/metabase/metabase/pull/48882/commits/e09b6ff5088a52a98cdbf388b1d1241c33fdb323